src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.product-card {
+  width: 200px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+
+  &__img {
+    height: 134px;
+    width: 160px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    margin: 0 0 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-color;
+  }
+
+  &__sku {
+    margin: 0 0 16px;
+    font-size: 10px;
+    font-weight: 400;
```

Comment:
these properties are duplicated several times. Can we set it as the default font property for the whole page and only change it when we need it?